toolkits/global/packages/global-forms/README.md

```diff
@@ -13,3 +17,265 @@ The `global-forms` component currently uses the `DEFAULT` brand only.
+
+```json
+"errorSummary": {
+    "id": "summary",
+    "title": "There are problems",
+    "errors": [
+        {
+            "id": "your-name-error",
+            "error": "Enter your name"
+        },
+        {
+            "id": "checkbox-terms-error",
+            "error": "Agree to the terms to continue"
+        }
+    ]
+}
+```
+
+### Making choices
+
+`Select` fields (using the `<select>` element) define their choices via an `options` property, which must be an array. The `selected` property is a Boolean:
+
+```json
+"options": [
+    {
+        "label": "Horse",
+        "value": "Horse"
+    },
+    {
+        "label": "Platypus",
+        "value": "Platypus",
+        "selected": true
+    }
+]
+```
+
+Radios define choices with an `inputs` array:
+
+```json
+{
+    "template": "radios",
+    "label": "Animal",
+    "inputs": [
+        {
+            "label": "Horse",
+            "value": "Horse",
+            "name": "animal"
+        },
+        {
+            "label": "Monkey",
+            "value": "Monkey",
+            "name": "animal"
+        }
+    ]
+}
+```
+
+The `name` property is placed at the top level and inherited by each input.
+
+Radios are always grouped together in a `fieldset`, so the group label (“Animal”, here) renders as a `legend` not a `label`.
+
+You might want to show users an additional field when they select a radio. For example, revealing a text input field for them to give more specific information about the option they’ve selected. 
+
+These fields can have any properties of a standard form field. Set these properties using the `fields` property (an array), like this:
+
+```json
+{
+    "label": "Monkey",
+    "value": "Monkey",
+    "name": "animal",
+    "fields": [
+        {
+            "template": "text",
+            "label": "What type of monkey?",
+            "id": "monkey-type",
+            "name": "monkey-type",
+            "hint": "Chimps are not monkeys"
+        }
+    ]
+}
+```
+
+Unlike radios, which are always used in sets of two or more, you can have a single checkbox field. To give users a set of checkbox choices, organise the checkboxes into a `fieldset`:
+
+```json
+{
+    "legend": "<h2>Which animals do you like?</h2>",
+    "fields": [
+        {
+            "template": "checkbox",
+            "label": "Monkeys",
+            "id": "checkbox-monkeys",
+            "name": "checkbox-monkeys"
+        },
+        {
+            "template": "checkbox",
+            "label": "Horses",
+            "id": "checkbox-horses",
+            "name": "checkbox-horses"
+        }
+    ]
+}
+```
+
+### Buttons
+
+A `template` of `buttons` defines a set of button controls, displayed inline (using Flexbox and `gap` for tidy wrapping). 
+The `type` property for each individual button corresponds to the `type` property/attribute. For example, here is how you would include a submit button:
```

Comment:
Can we make a choice here between property and attribute, and lose the slash? If it really could be either, can we say "or"?